### PR TITLE
Upgrade Django to 1.11 [One Step at a time]

### PR DESCRIPTION
**Pull request by aktech**
Upgrading everything to latest version in one go is not a great idea and will be hard to review as well, I am planning to upgrading things slowly and one by one so that the diff is not a lot.

- [x] This PR upgrades Django (1.3 -> 1.11)
- [x] Minor pep8 changes
- [x] Update copyright to 2020 (making it automatic is one for later)
- [x] Tests Pass

@asmeurer @certik @lidavidm

### CODE REVIEW

File: templates/base.html [hunk 1]
```diff
@@ -64,7 +64,7 @@
         </p>
 
         <p>
-        &copy; 2013 SymPy Development Team. This project is
+        &copy; 2020 SymPy Development Team. This project is
```
Comment (asmeurer) [inline templates/base.html:67]:
```suggestion
        &copy; 2013-2020 SymPy Development Team. This project is
```

**Comment by asmeurer** [comment]
@prshnt19 has done some work with Django for SymPy Live https://github.com/sympy/sympy-live/pull/144. Perhaps you can review each other's work.

I can also give you access to push up to the App Engine if you need it.

**Comment by aktech** [comment]
@asmeurer Sure, sounds great!

**Comment by aktech** [comment]
@prshnt19 Can you please have a look at this PR?

File: app/views.py [hunk 2]
```diff
@@ -279,19 +279,19 @@ def eval_card(request, card_name):
     except ValueError as e:
         return HttpResponse(json.dumps({
             'error': e.message
-        }), mimetype="application/json")
+        }), content_type="application/json")
     except DeadlineExceededError:
         return HttpResponse(json.dumps({
             'error': 'Computation timed out.'
-        }), mimetype="application/json")
+        }), content_type="application/json")
```
Comment (prshnt19) [inline app/views.py:286]: I think its better to use JsonResponse.

File: app/views.py [hunk 3]
```diff
@@ -381,12 +382,14 @@ def remove_query(request, qid):
             'message': 'Not logged in or invalid user.'
         }
 
-    return HttpResponse(json.dumps(response), mimetype='application/json')
+    return HttpResponse(json.dumps(response), content_type='application/json')
```
Comment (prshnt19) [inline app/views.py:385]:
```suggestion
    return JsonResponse(response)
```

File: settings.py [hunk 4]
```diff
@@ -70,9 +63,21 @@
...
+            ('django.template.loaders.cached.Loader', [
+                'django.template.loaders.filesystem.Loader',
+                'django.template.loaders.app_directories.Loader',
+            ]),
+        ],
+    },
+}]
+
+
+ALLOWED_HOSTS = ['127.0.0.1', 'localhost']
```
Comment (prshnt19) [inline settings.py:80]: Allowed hosts should contain the domain https://sympygamma.com for production.

File: app.yaml.template [hunk 5]
```diff
@@ -26,6 +26,8 @@ handlers:
 
 libraries:
 - name: django
-  version: "1.3"
+  version: "1.11"
 - name: numpy
   version: "1.6.1"
```
Comment (prshnt19) [inline app.yaml.template:31]:
```suggestion
  version: "1.18.1"
```
I think numpy should be updated to latest.

**Comment by aktech** [comment]
> @aktech I think instead of Python2 we should use Python3. Python2 support is already dropped for SymPy.

@prshnt19 Thanks for the review. This is beyond the scope of this PR. The plan is to do incremental working updates. See [this comment](https://github.com/sympy/sympy_gamma/pull/135#issue-402065889). This PR only contains minimum amount of changes required to support Django 1.11, following PR could contain upgrading further versions incrementally.

File: app/views.py (hunk 3 above)
Comment (aktech) [reply app/views.py:385]: Beyond the scope, [See this](https://github.com/sympy/sympy_gamma/pull/135#issuecomment-612584314)

File: app.yaml.template (hunk 5 above)
Comment (aktech) [reply app.yaml.template:31]: Beyond the scope, [See this](https://github.com/sympy/sympy_gamma/pull/135#issuecomment-612584314)

File: app/views.py (hunk 2 above)
Comment (aktech) [reply app/views.py:286]: Beyond the scope, [See this](https://github.com/sympy/sympy_gamma/pull/135#issuecomment-612584314)

**Comment by prshnt19** [comment]
Well, I only have doubts for python2. Other than that changes look fine to me

**Comment by aktech** [comment]
> Well, I only have doubts for python2. Other than that changes look fine to me

This is an attempt to prepare for Python3 upgrade. It will happen incrementally.

**Comment by asmeurer** [comment]
SymPy Gamma is also hosted at gamma.sympy.org

**Comment by aktech** [comment]
> SymPy Gamma is also hosted at gamma.sympy.org

Updated `ALLOWED_HOSTS` and deployed here:
https://sympy-gamma-aktech.uc.r.appspot.com/input/?i=x

**Pull request merged**